go/vt/mysqlctl/s3backupstorage/retryer.go

```diff
@@ -0,0 +1,39 @@
+package s3backupstorage
+
+import (
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/request"
+)
+
+type ClosedConnectionRetryer struct {
```

Comment:
Looks like this struct satisfies a particular interface. That should be documented.